ROCm_Tools/ROCm-Tools.rst

```diff
@@ -464,201 +464,244 @@ References
        *  `AMD GCN3 Instruction Set Architecture (2016) <http://developer.amd.com/wordpress/media/2013/12/AMD_GCN3_Instruction_Set_Architecture_rev1.1.pdf>`_
        *  `AMD_Southern_Islands_Instruction_Set_Architecture <https://developer.amd.com/wordpress/media/2012/12/AMD_Southern_Islands_Instruction_Set_Architecture.pdf>`_
 
-ROCm-GDB
-=========
+ROCr Debug Agent
+================
 
-The ROCm-GDB repository includes the source code for ROCm-GDB. ROCm-GDB is a modified version of GDB 7.11 that supports debugging GPU kernels on Radeon Open Compute platforms (ROCm).
+The ROCr Debug Agent is a library that can be loaded by ROCm Platform
+Runtime to provide the following functionality:
 
-Package Contents
-##################
-The ROCm-GDB repository includes
+-  Print the state of wavefronts that report memory violation or upon
+   executing a ``s_trap 2`` instruction.
+-  Allows SIGINT (``ctrl c``) or SIGTERM (``kill -15``) to print
+   wavefront state of aborted GPU dispatches.
+-  It is enabled on Vega10 GPUs on ROCm1.9.
 
-   * A modified version of gdb-7.11 to support GPU debugging. Note the main ROCm specific files are located in gdb-7.11/gdb with the 	  rocm-* prefix.
-   * The ROCm debug facilities library located in amd/HwDbgFacilities/. This library provides symbol processing for GPU kernels.
+Usage
+*****
 
-Build Steps
-############
- 
-1. Clone the ROCm-GDB repository
+To use the ROCr Debug Agent set the following environment variable:
 
-::
-   
-    git clone https://github.com/RadeonOpenCompute/ROCm-GDB.git
+.. code:: sh
 
-2. The gdb build has been modified with new files and configure settings to enable GPU debugging. The scripts below should be run to 	  compile gdb. The run_configure_rocm.sh script calls the GNU autotools configure with additional parameters. The   	 	    	run_configure_rocm.sh script will create the build directory to build the gdb executable in a out of source manner
+   export HSA_TOOLS_LIB=librocr_debug_agent64.so
 
-::
+This will use the ROCr Debug Agent library installed at
+/opt/rocm/lib/librocr_debug_agent64.so by default since the ROCm
+installation adds /opt/rocm/lib to the system library path. To use a
+different version set the LD_LIBRARY_PATH, for example:
 
-    ./run_configure_rocm.sh debug
+.. code:: sh
 
-3.    The run_configure_rocm.sh script also generates the run_make_rocm.sh which sets environment variables for the Make step
+   export LD_LIBRARY_PATH=/path_to_directory_containing_librocr_debug_agent64.so
 
-::
-   
-   ./run_make_rocm.sh
+To display the machine code instructions of wavefronts, together with
+the source text location, the ROCr Debug Agent using the llvm-objdump
+tool. Ensure that a version that supports AMD GCN GPUs is on your
+’$PATH`. For example, for the ROCm1.9:
```

Comment:
Small grammar changes requested, since this is documentation. :)

"using the llvm-objdump" --> "uses the llvm-objdump"
"for the ROCm1.9" --> "for ROCm 1.9"

Reply:
Other than this, commits looks great to me!

Reply:
Made the changes. Thanks.